bin/stashcp2/tests/test.sh

```diff
@@ -0,0 +1,4 @@
+python stashcp.py --debug user/taburaad/public/2gb_file.tar .
```

Comment:
I don't think this will work since xrdcp is not installed by default at travis-ci.  I think you will need something like what we do with the HTCondor-CE, where we [start a CentOS docker image](https://github.com/opensciencegrid/htcondor-ce/tree/master/tests).

1. Setup a [docker image](https://github.com/opensciencegrid/htcondor-ce/blob/master/.travis.yml) in the travis-ci.
1. [Create the docker image](https://github.com/opensciencegrid/htcondor-ce/blob/master/tests/setup_tests.sh).
1. Finally, [run the tests](https://github.com/opensciencegrid/htcondor-ce/blob/master/tests/test_inside_docker.sh)